### DISCUSSION TITLE
请问如何自定义几个emoji?

### DISCUSSION THREAD

**Discussion by rsyqvthv** [rime/home#1330] [Q&A]
日常输入的时候我只需要2个emoji: ❌ 和 ✔️

请问如何设置可以在输入 xx后, 上屏 ❌, 输入 rr后, 上屏 ✔️?

我不需要配置全套的emoji, 只需要这两个就满足我的需要了. 

感谢.

**Comment by LEOYoon-Tsaw** [comment]
加進字典

**Reply by rsyqvthv** [reply]
请问要怎么加入字典呢？

我会加一些非emoji的自定义词，比如人名什么的，但同样的方式用在emoji上就不起作用，所以想问您具体是怎么做的？

感谢

**Reply by rsyqvthv** [reply]
补充下, 我的设置文件是这样的, 前两个可以上屏, 后两个(xx, rr)就没有相应的候选.

文件: aurora_pinyin.extended.dict.yaml

```
---
name: aurora_pinyin.extended
version: "2021/09/22"
sort: by_weight
use_preset_vocabulary: true
import_tables:
  - aurora_pinyin

^_^	hh	5
T_T	kk	5

❌	xx	5
✔️	rr	5
````
